build/Nuget-prep.ps1

```diff
@@ -0,0 +1,36 @@
+# Copy files needed for Nuget package generation to appropriate folder
+
+$RootPath = Split-Path $PSScriptRoot -Parent
+$SceneLoader = $RootPath + '\SceneLoader\'
+$lib = $RootPath + '\SceneLoader\NugetPackager\lib\uap10.0\'
```

Comment:
> $RootPath + '\SceneLoade [](start = 6, length = 25)

Don't keep repeating the same subpaths. You created a variable for $SceneLoader but you're not using it. You should also create a subpath for $SceneLoader\NugetPackager.
And use the PowerShell syntax:   $SceneLoader ="$RootPath\SceneLoader" rather than the + operator... much easier to read for paths.